### ISSUE TITLE
permissive license

### ISSUE THREAD

**Issue by bvanevery** [gongminmin/KlayGE#198]
I hope you will consider a permissive license instead of the GPL. Like MIT, BSD, Apache, or zlib. The GPL is not friendly to struggling indie game developers who would someday like to make money with what they work on. There's basically no endgame where one can pay the bills. There are a lot of game engines out there and permissive licenses are very common.

**Comment by ProtectedVariable** [comment]
Agreed, another good option is LGPL, as it would allow games to make profit but improvements to the engine itself need to be open source

**Comment by WonderCooper** [comment]
您好，已收到您的邮件，我会尽快处理，谢谢
